MonMaraicher/Features/Map/FarmerService.swift

```diff
@@ -9,26 +9,33 @@ import Foundation
 
 final class FarmerService {
 
-    func loadFarmers(forName ressourceName: String) throws -> [Farmer] {
-        let decoder = JSONDecoder()
-        decoder.keyDecodingStrategy = .convertFromSnakeCase
+    func loadFarmers() async throws -> Farmers {
```

Comment:
Nice one! You are catching errors and use async/await so good job. Now I can be improved to be unit tested. You can try it or we can see together how to do it.

The issue here is how would you do if you need to fetch a 2nde request? Will you copy paste the code here and change the URL?